tmt/steps/provision/podman.py

```diff
@@ -283,6 +284,33 @@ def _run_ansible(
             log=log,
             silent=silent)
 
+    @functools.cached_property
+    def _is_toolbox(self) -> bool:
+        """ Return ``True`` if running in toolbox, ``False`` otherwise. """
+        if os.path.exists('/run/.toolboxenv'):
+            return True
+
+        return False
+
+    @functools.cached_property
+    def _toolbox_container_name(self) -> Optional[str]:
+        """ Returns toolbox container name if running in toolbox, ``None`` otherwise. """
+        if not self._is_toolbox:
+            return None
+
+        if not os.path.exists('/run/.containerenv'):
+            self.logger.warning(
+                "Unable to detect toolbox container name, '/run/.containerenv' not found."
+                )
+            return None
+
+        with open('/run/.containerenv') as envfile:
+            for line in envfile.readline():
+                if line.startswith('name="'):
+                    return line[6:-1]
+
+        return None
```

Comment:
These two look like they belong to guest facts, as they are properties of a machine similar to `has_selinux` or `arch`, for example. Once moved, you would reach them via  `self.parent.plan.my_run.runner.facts.*`.

Reply:
thanks for the pointers

Reply:
moved to facts